chromiumoxide_pdl/src/build/generator.rs

```diff
@@ -213,15 +213,24 @@ impl Generator {
             modules.extend(module);
         }
 
-        // fix unresolved type sizes
-        let mut refs = Vec::new();
+        // brute-force fix unresolved type sizes
+        let mut refs = VecDeque::new();
         std::mem::swap(&mut refs, &mut self.ref_sizes);
```

Comment:
Now that I'm reading this again I think [`std::mem::replace`](https://doc.rust-lang.org/std/mem/fn.replace.html) would be more idiomatic here.

Reply:
`std::mem::replace` requires that the `src` (second arg) is not a reference, but `self` is `&mut` here, so it's not possible to use I believe.